examples/knative/go/kn-go-preemption/function.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-go-preemption
+  labels:
+    app: veba-ui
+    workflow: vsphere-preemption
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: "1"
+        autoscaling.knative.dev/minScale: "1"
+    spec:
+      containers:
+        - image: us.gcr.io/daisy-284300/veba/kn-go-preemption:1.0
+          env:
+            - name: TEMPORAL_URL
+              value: "FILL-ME-IN"
+            - name: TEMPORAL_NAMESPACE
+              value: "FILL-ME-IN"
+            - name: TEMPORAL_TASKQUEUE
+              value: "FILL-ME-IN"
+            - name: VSPHERE_PREEMPTION_TAG
+              value: "FILL-ME-IN"
+            - name: VSPHERE_ALARM_NAME
+              value: "FILL-ME-IN"
```

Comment:
nit: Could we use `FILL_ME_IN` instead. I started to do that as I realize, you can't easily click on the variable and replace text because of the "dash" :)

Reply:
good point, will do